tests/cli-v2.rs

```diff
@@ -73,6 +74,39 @@ fn install_toolchain_from_version() {
     });
 }
 
+#[test]
+fn install_with_profile() {
+    clitools::setup(Scenario::Full, &|config| {
```

Comment:
The scenario in use in `cli-v2.rs`'s `setup_complex` is likely sufficient and already available.  While it's not *bad* to use `clitools::setup()` here, it is at variance with the usual practice in this test file.  Having the setup functions all in one place makes it easier if we need to do something "interesting" in order to make sure Windows behaves.

Reply:
I mean `setup_complex` is only used by one test, but sure :p